src/content/4/en/part4b.md

```diff
@@ -723,6 +727,19 @@ test('a note can be deleted', async () => {
   expect(contents).not.toContain(noteToDelete.content)
 })
 ```
+In the test for 'a specific note can be viewed' we get an error because like the ID in 3b the date is stored as an Object in Mongoose without extra formatting but is compared to a JSON string.
+
+Let's add this to our models/note.js
+
+```js
+noteSchema.set('toJSON', {
+  transform: (document, returnedObject) => {
+  returnedObject.id = returnedObject._id.toString()
+  returnedObject.date = returnedObject.date.toString() // highlight-line
```

Comment:
I would rely on the `new Date().toJSON()` implementation here, which is `new Date().toISOString()`

Reply:
looks less hacky than my attempt at fixing it with the knowledge of the course material :)